LEGO1/mxomni.cpp

```diff
@@ -248,8 +344,24 @@ void MxOmni::Destroy()
 // OFFSET: LEGO1 0x100b07f0
 MxLong MxOmni::Notify(MxParam &p)
 {
-  // FIXME: Stub
-  return 0;
+  MxAutoLocker lock(&this->m_criticalsection);
+  MxLong result;
+  if (p.GetType() == 2)
```

Comment:
I think it would be a good moment to start introducing an `enum` in `MxParam` to have named constants for the known (and unknown) notification types, along the lines of: https://github.com/isledecomp/isle/blob/master/LEGO1/mxpresenter.h#L18

and also use it as the type for `m_type` in `MxParam`.